homeassistant/components/fibaro/climate.py

```diff
@@ -225,7 +237,7 @@ def set_fan_mode(self, fan_mode):
     def fibaro_op_mode(self):
         """Return the operating mode of the device."""
         if not self._op_mode_device:
-            return 6  # Fan only
+            return 3  # Default to AUTO
```

Comment:
@pbalogh77 just want to make sure that mode is correct, does this mean that device is handling its own schedule?

Thats what HA means by auto mode.

https://developers.home-assistant.io/docs/core/entity/climate/#hvac-modes

Reply:
I think that's the closest it gets to, so yes.

Reply:
I think that's the closest it gets to, so yes.

Reply:
Remember that there is a device and there's fibaro home center. So it could get its smarts from multiple places. All we know for sure is that this device has not exposed to us a way to get or set its operating mode, only its target temperature usually. So we don't know whether it's a heating, cooling or a fan device. All we know it exists and we can set a temperature target to it and it'll somehow do something with it.